common/production_types.txt

```diff
@@ -477,35 +477,35 @@ stables = {
 artisinal_workshop = {
 	template = RGO_template_clerks
 	output_goods = artisinal_goods
-	value = 1
+	value = 3.9
```

Comment:
Is this intentional? Artisinal goods is not supposed to be as profitable as precious metals. Right off the start I can see that the labourers are filling all of their life, most of their everyday and some of their luxury needs, and the nobles are right there with them too